src/ucs/memory/rcache.c

```diff
@@ -423,11 +423,10 @@ static void ucs_rcache_region_invalidate(ucs_rcache_t *rcache,
                                    ucs_status_string(status));
         }
         region->flags &= ~UCS_RCACHE_REGION_FLAG_PGTABLE;
+        ucs_rcache_region_put_internal(rcache, region, flags);
```

Comment:
can you pls add gtest? (i guess that invalidate on the region not in pgtable should not fail)?